src/Components/DetailItem/DropDownBox.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+import './DropDownBox.scss';
+import OrderBox from './OrderBox';
+
+class DropDownBox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      number: 0,
+      value: '',
+      size: '',
+      selectPrice: 16200,
+      defaultPrice: 0,
+      click: false,
+      click2: false,
+      item: [],
+      orderBox: [],
+      priceBox: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({});
+  }
+  // 상품 클릭시 클릭 바뀌게
+  handleChange = (el) => {
+    const { click, number } = this.state;
+    this.setState({ number: number + 1, value: el.label });
+  };
+
+  //사이즈 클릭시 값이 누적되고 오더박스 카드가 생김
+  handleSizeCheck = (e) => {
+    // console.log('e는 뭐지?', e);
+    const { click2, selectPrice, item, value, click, orderBox } = this.state;
+    let arr = [...item];
+    arr.push(selectPrice);
+    // console.log('arr에 item[] 복사됨 : ', arr);
+    let totalPrice = arr.reduce((acc, obj) => acc + obj, 0);
```

Comment:
reduce 활용 굿굿!! ☺️ 아래와 같이 사용할 수 있겠네요. 
```suggestion
    let totalPrice = arr.reduce((acc, obj) => acc + obj);
```

Reply:
지금도 코드가 바껴서 저 코드랑은 달라졌습니다 나중에 응용해보겠습니다 :)